### PR TITLE
Hadiism Versebook

### PR DESCRIPTION
**Pull request by Ben10083**
- [x] Code
- [ ] LORE (desc and .txt file verses
- [x] Sprite

New versebook: Hadiist Manifesto. Replacing the old book

### CODE REVIEW

**Comment by Ben10083** [comment]
!wip

File: code/game/objects/items/devices/versebook.dm [hunk 1]
```diff
@@ -71,6 +71,17 @@
 	. = ..()
 	randomquip = file2list("ingame_manuals/biesel.txt")
 
+/obj/item/device/versebook/pra
+	name = "\improper Hadiist Manifesto"
+	desc = "A compact red book that is commonly considerd required reading for PRA citizens. This book can be found anywhere the PRA is, with the party commonly providing free copies of the Manifesto to tourists."
```
Comment (Generalcamo) [inline code/game/objects/items/devices/versebook.dm:76]:
```suggestion
	desc = "A compact red book that is commonly considered required reading for PRA citizens. This book can be found anywhere the PRA is, with the party commonly providing free copies of the Manifesto to tourists."
```

File: code/game/objects/items/devices/versebook.dm [hunk 2]
```diff
@@ -61,7 +61,7 @@
 	randomquip = file2list("ingame_manuals/gadpathur.txt")
 
 /obj/item/device/versebook/biesel
-	name = "\improper Constitution of The Federal Republic of Biesel"
+	name = "\improper Constitution of The Federal Republic of Biesel (abdridged)"
```
Comment (Generalcamo) [inline code/game/objects/items/devices/versebook.dm:64]:
```suggestion
	name = "\improper Constitution of The Federal Republic of Biesel (abridged)"
```

File: code/game/objects/items/devices/versebook.dm [hunk 3]
```diff
@@ -71,6 +71,17 @@
 	. = ..()
 	randomquip = file2list("ingame_manuals/biesel.txt")
 
+/obj/item/device/versebook/pra
+	name = "\improper Hadiist Manifesto"
+	desc = "A compact red book that is commonly considerd required reading for PRA citizens. This book can be found anywhere the PRA is, with the party commonly providing free copies of the Manifesto to tourists."
+	desc_extended = "HE/SHE LOVES HADII."
```
Comment (Generalcamo) [inline code/game/objects/items/devices/versebook.dm:77]: This is unlikely to be accepted.

**Review by Generalcamo — CHANGES_REQUESTED** [review]

File: code/game/objects/items/devices/versebook.dm (hunk 3 above)
Comment (Ben10083) [reply code/game/objects/items/devices/versebook.dm:77]: It would

**Comment by FluffyGhoster** [comment]
Merge master to fix the CI

**Comment by Ben10083** [comment]
Didn't mean to mark it. Oops

**Comment by Ben10083** [comment]
This will be closed in favor of a newer version with a wider scope and less conflicts.

**Comment by Ben10083** [comment]
https://github.com/Aurorastation/Aurora.3/pull/19870

**Pull request closed without merging**